src/config/interfaces/Config/IURLConfig.ts

```diff
@@ -54,10 +69,29 @@ interface IURLConfig {
    */
   analytics?: boolean;
 
+  /**
+   * Whether or not to sign the URL
+   */
   signUrl?: boolean;
+
+  /**
+   * Whether or not to use a long signature
+   */
   longUrlSignature?: boolean;
+
+  /**
+   * Whether or not to shorten the URL
+   */
   shorten?: boolean;
+
+  /**
+   * Whether or not to use the root path
+   */
   useRootPath?: boolean;
+
+  /**
+   * Whether or not to force a version
+   */
```

Comment:
I think that some of these need more explanation. Maybe just some examples would help. What does it mean to shorten the URL? What's the root path? What does it mean to force a version?

Reply:
I've linked this whole file (at the top) to the config params with a learn more to avoid adding mountains of texts to explain each and every config.

Consider how complicated it is to expalin the difference between `secureDistribution`, `secure`, and `cname`.

Reply:
@carlevison I added an internal ticket to extend these in time, once we finish with other priorities